src/assignments/hello-world.rst

```diff
@@ -1,4 +1,65 @@
-.. _hello-world:
+.. _assignment0:
 
 Assignment #0: Hello, World!
-============================
\ No newline at end of file
+============================
+
+The purpose of this initial assignment is to familiarize yourself with the process of running the autograding tests and submitting your work via GitHub Classroom. Even if you are familiar with GitHub Classroom, this assignment will contain several new items that are specific to working with Java assignments.
+
+Getting Started
+---------------
+
+Let's set up the assignment on our computer and learn about its basic structure.
+
+First, find Assignment #0 in Canvas and click on the invitation link. After accepting, go to your assignment repository on GitHub.
```

Comment:
GitHub Classroom?

I feel like we should make a distinction between GH and GHClassroom for clarification and consistency.  It will hopefully help students look in the right place for starter code and submit things correctly.

Reply:
Yeah, they will almost never actually see Github Classroom, but I tried to add some clarity here! Thanks!